From 6923e8fb56692b20d24398d4746d2399490acdc1 Mon Sep 17 00:00:00 2001
From: Leonardo Brondani Schenkel <leonardo.schenkel@gmail.com>
Date: Mon, 3 Oct 2016 10:47:42 +0200
Subject: Fix link of p11-kit-proxy.dylib on Mac OS X

However, on Mac OS X the library is named libp11-kit.dylib so
in the above command the source of the link resolves to nothing,
the destination becomes the source and the link to a non-existent
file is created in the working directory.

https://bugs.freedesktop.org/show_bug.cgi?id=98022

diff --git a/p11-kit/Makefile.am b/p11-kit/Makefile.am
index 3ef70f9..62cf70d 100644
--- a/p11-kit/Makefile.am
+++ b/p11-kit/Makefile.am
@@ -87,7 +87,7 @@ libp11_kit_testable_la_CFLAGS = \
 
 # Proxy module is actually same as library, so install a link
 install-exec-hook:
-	$(LN_S) -f `readlink $(DESTDIR)$(libdir)/libp11-kit.so` $(DESTDIR)$(libdir)/p11-kit-proxy.so
+	$(LN_S) -f `readlink $(DESTDIR)$(libdir)/libp11-kit.{so,dylib}` $(DESTDIR)$(libdir)/p11-kit-proxy.so
 	$(MKDIR_P) $(DESTDIR)$(p11_package_config_modules)
 
 endif
-- 
cgit v0.10.2

